crates/scarb-api/src/lib.rs

```diff
@@ -51,7 +51,19 @@ impl StarknetContractArtifacts {
         let sierra = fs::read_to_string(sierra_path)?;
 
         let casm = match &starknet_contract.artifacts.casm {
-            None => String::new(),
+            None => {
+                let casm = Command::new("universal-sierra-compiler")
+                    .current_dir(base_path)
+                    .args(vec![
+                        "--sierra-input-path",
+                        starknet_contract.artifacts.sierra.as_str(),
+                    ])
+                    .output()
+                    .unwrap()
```

Comment:
Maybe `context` informing what has happened and encouraging to:
1. Make sure you have the latest universal compiler binary installed (e.g. sierra 1.5 is coming out soon, it won't work with previous compiler binaries).
2. Contact us if it doesn't help (Unexpected Protostar error vibes xD), since this mean for 99% that we have fucked up and need to fix it asap